### PR TITLE
Format value before comparission in number.js

### PR DESCRIPTION
**Pull request by krlakwa**

### CODE REVIEW

File: src/alto-ui/helpers/number.js [hunk 1]
```diff
@@ -18,10 +18,10 @@ const DEFAULT_DECIMAL_SEPARATOR = '.';
 export const round = (n, precision) => Math.round(n * 10 ** precision) / 10 ** precision;
 
 export const parse = (value, locale, precision = 0) => {
-  const parsedValue = parseFloat(value);
-
   const decimalSeparator = decimalSeparatorByLocale[locale] || DEFAULT_DECIMAL_SEPARATOR;
 
+  const parsedValue = parseFloat(value);
```
Comment (NicolasMartos) [inline src/alto-ui/helpers/number.js]: Why this modification ?

Comment (krlakwa) [reply src/alto-ui/helpers/number.js]: otherwise, it incorrectly formats the value with decimal places

Comment (renaudtertrais) [reply src/alto-ui/helpers/number.js]: can you update you PR then ?

Comment (krlakwa) [reply src/alto-ui/helpers/number.js]: yes, I'll change it back

**Comment by renaudtertrais** [comment]
Wrong fix.

**Pull request closed without merging**